tests/source/macros.rs

```diff
@@ -34,6 +34,13 @@ fn main() {
     a,   
     ];
     
+    vec![a; b];
+    vec!(a; b);
+    vec!{a; b};
+
+    vec![a, b; c];
+    vec![a; b, c];
```

Comment:
I added a few tests. Could you take a look if they suffices or there should be more tests?